src/main/java/fluke/Fluke.java

```diff
@@ -0,0 +1,212 @@
+     */
+    private void addDeadline(String command) throws FlukeException {
+        String[] parsedCommand = Parser.parseDeadlineCommand(command);
+        String description = parsedCommand[0];
+        String byDate = parsedCommand[1];
+        Task taskAdded = this.tasks.addDeadline(description, byDate);
+        this.ui.showTaskAdded(taskAdded, tasks);
+    }
+
+    /**
+     * Tells Fluke to add an Event to its tasks.
+     * @param command Command given to Fluke.
+     * @throws FlukeException if an error occurs while adding the task, for instance, with invalid inputs.
+     */
+    private void addEvent(String command) throws FlukeException {
+        String[] parsedCommand = Parser.parseEventCommand(command);
+        String description = parsedCommand[0];
+        String fromDate = parsedCommand[1];
+        String toDate = parsedCommand[2];
+        Task taskAdded = this.tasks.addEvent(description, fromDate, toDate);
+        this.ui.showTaskAdded(taskAdded, tasks);
+    }
+
+    /**
+     * Tells Fluke to mark a task as done.
+     * @param command Command given to Fluke.
+     * @throws FlukeException if an error occurs while marking a task as done, for instance, with invalid inputs.
+     */
+    private void markTaskAsDone(String command) throws FlukeException {
+        int index = Parser.parseMarkAsDoneCommand(command);
+        Task taskMarked = tasks.markTaskAsDone(index);
+        this.ui.showTaskMarkedAsDone(taskMarked);
+    }
+
+    /**
+     * Tells Fluke to mark a task as not done.
+     * @param command Command given to Fluke.
+     * @throws FlukeException if an error occurs while marking a task as not done, for instance, with invalid inputs.
+     */
+    private void markTaskAsUndone(String command) throws FlukeException {
+        int index = Parser.parseMarkAsUndoneCommand(command);
+        Task taskMarked = tasks.markTaskAsUndone(index);
+        this.ui.showTaskMarkedAsUndone(taskMarked);
+    }
+
+    /**
+     * Tells Fluke to delete a task.
+     * @param command Command given to Fluke.
+     * @throws FlukeException if an error occurs while deleting a task, for instance, with invalid inputs.
+     */
+    private void deleteTask(String command) throws FlukeException {
+        int index = Parser.parseDeleteCommand(command);
+        Task deleted = tasks.deleteTask(index);
+        this.ui.showTaskDeleted(deleted, tasks);
+    }
+
+    /**
+     * Fluke attempts to find tasks with the corresponding keyword.
+     * @param command the command given
+     * @throws InvalidInputException if there is no keyword given.
+     */
+    private void findTask(String command) throws InvalidInputException {
+        String keyword = Parser.parseFindCommand(command);
+        this.ui.showTasksWithKeyword(this.tasks.findTask(keyword));
+    }
+
+    /**
+     * Helper function for additional logic related to changing the list.
```

Comment:
Nice use of commenting to describe each method and what it does. Perhaps add a new line between the description and the params to fit the coding standard.